dnf-behave-tests/features/help.feature

```diff
@@ -26,3 +26,51 @@ Scenario: Command help
    When I execute dnf with args "install --unknown-option"
    Then the exit code is 2
    And stderr contains ".+ install: err"
+
+
+Scenario Outline: Help should refer to yum/dnf depending on what command was used
+   When I execute dnf with args "<args>"
+   Then the exit code is 0
+    And stdout contains "usage: {context.dnf.dnf_command}"
+
+Examples:
+        | args      |
+        | help      |
+        | --help    |
+
+
+@use.with_os=rhel__ge__8
+@use.with_os=centos__ge__8
+@use.with_os=fedora__ge__30
+@use.with_os=fedora__lt__30
+Scenario Outline: Help should refer to yum/dnf depending on what command was used (explicitly testing "dnf" and "yum")
+   When I execute "<command>" with args "<args>"
+   Then the exit code is 0
+    And stdout contains "usage: <command>"
+
+Examples:
+        | args      | command  |
+        | help      | dnf      |
+        | --help    | dnf      |
+        | help      | dnf-3    |
+        | --help    | dnf-3    |
+        | help      | yum      |
+        | --help    | yum      |
```

Comment:
Maybe flip the two columns for a bit better readability?

Reply:
Done.